semantic-conventions/setup.cfg

```diff
@@ -36,10 +36,10 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    dataclasses~=0.6
+    dataclasses~=0.6;python_version<'3.7'
```

Comment:
I don't think we support python < 3.7 at all, probably we need the Python version we currently run with in the container as minimum.